serial_tests/test_serial.py

```diff
@@ -0,0 +1,136 @@
+import queue
+import random
+import socket
+import threading
+
+from dataclasses import dataclass
+from typing import Tuple
+
+import pytest
```

Comment:
Please sort imports

Reply:
Done in 662a21c
Sorted imports alphabetically, then from ... imports, and finally our own from RCTComms import